### PR TITLE
Feature/symfony 5 upgrade

### PR DESCRIPTION
**Pull request by turegjorup**
#### Link to ticket

No ticket

#### Description

Tested upgrade to Symfony 5

#### Screenshot of the result

No screenshot

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -10,8 +10,8 @@ return PhpCsFixer\Config::create()
     ->setRules([
         '@Symfony' => true,
         '@Symfony:risky' => false,
-        'phpdoc_align' => false,
-        'no_superfluous_phpdoc_tags' => false,
+        'phpdoc_align' => true  ,
+        'no_superfluous_phpdoc_tags' => true,
```
Comment (cableman) [inline .php_cs.dist:14]: Why the align? It looks not good.

File: public/index.php [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
     Debug::enable();
 }
 
-if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? $_ENV['TRUSTED_PROXIES'] ?? false) {
-    Request::setTrustedProxies(explode(',', $trustedProxies), Request::HEADER_X_FORWARDED_ALL ^ Request::HEADER_X_FORWARDED_HOST);
-}
-
-if ($trustedHosts = $_SERVER['TRUSTED_HOSTS'] ?? $_ENV['TRUSTED_HOSTS'] ?? false) {
-    Request::setTrustedHosts([$trustedHosts]);
-}
```
Comment (cableman) [inline public/index.php:21]: We use the trusted proxy to get the right ip inside the cluster?

File: .php_cs.dist (hunk 1 above)
Comment (cableman) [reply .php_cs.dist:14]: I think it almost make the code readable... please switch back

**Comment by cableman** [comment]
Why the reactor files?

File: public/index.php (hunk 2 above)
Comment (turegjorup) [reply public/index.php:21]: This is copy paste from https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/5.2/public/index.php

I had to manually update the files that are installed by flex recipes to get things working. Changes here are not "my" changes, but changes to the official defaults. If there are changes to how trusted proxies are handled in 5.x I don't know.

File: .php_cs.dist (hunk 1 above)
Comment (turegjorup) [reply .php_cs.dist:14]: ### Symfony
```
    /**
     * Transforms the input given as first argument.
     *
     * @param bool|string $dummy   Some argument description
     * @param array       $options An options collection to be used within the transformation
     *
     * @return string|null The transformed input
     *
     * @throws \RuntimeException When an invalid option is provided
     */
    private function transformText($dummy, array $options = [])
    {
    ...
    }
```
See https://symfony.com/doc/current/contributing/code/standards.html

Comment (turegjorup) [reply .php_cs.dist:14]: > I think it almost make the code readable...

You're right - it almost does make the code readable 😄

Comment (turegjorup) [reply .php_cs.dist:14]: On a more serious note. It's possible that we can rector to understand "our" style but we should ask ourselves why we deviate from the established standards.

**Comment by turegjorup** [comment]
Changes to phpdoc reverted.

**Pull request closed without merging**